src/scipp/core/bin_remapping.py

```diff
@@ -95,8 +95,8 @@ def _combine_bins(
     # bins, without mixing contents from different unchanged bins.
     var = var.transpose(unchanged_dims + changed_dims)
     params = DataArray(var.bins.size(), coords=coords)
-    params.attrs['begin'] = var.bins.constituents['begin'].copy()
-    params.attrs['end'] = var.bins.constituents['end'].copy()
+    params.coords['begin'] = var.bins.constituents['begin'].copy()
+    params.coords['end'] = var.bins.constituents['end'].copy()
```

Comment:
Can these leak into the output? And should they?

Reply:
As far as I can tell `params` is not used directly?